dashboard/app/models/sections/section.rb

```diff
@@ -606,6 +606,11 @@ def assigned_csa?
     script&.csa? || [CSA, CSA_PILOT_FACILITATOR].include?(unit_group&.family_name)
   end
 
+  def assigned_gen_ai?
```

Comment:
Could you add 'allthethings' for local testing?

Reply:
Local testing can still be done via a levelbuilder account or a verified teacher account, are you looking for a way to get student accounts to have access?

Reply:
yes.

Reply:
mmm I think I'd prefer to limit access to our AWS resources as much as possible -- you can always remove this check locally if you'd like.

Reply:
Oh yeah - that sounds good and limiting makes sense.